addons/mail/static/src/views/web/fields/html_mail_field/html_mail_field.js

```diff
@@ -17,7 +17,15 @@ export class HtmlMailField extends HtmlField {
         }
         const cssRules = cssRulesByElement.get(editor.editable);
         el.classList.remove("odoo-editor-editable");
+        // Insert the cloned element inside an iframe so we can get its computed style.
+        const iframe = document.createElement('iframe');
+        iframe.style.height = '0px';
+        iframe.style.visibility = 'hidden';
+        iframe.setAttribute('sandbox', 'allow-same-origin'); // Make sure no scripts get executed.
+        iframe.appendChild(el);
```

Comment:
I don't get why we need to add `el` to an iframe and then add the iframe to the body. Why can't we add el to the body?

Isn't it an issue that your iframe does not have any css asset ? Will the computed style really be what we need without any css asset ? Or maybe it's the reason why you used the iframe in the first place, to not have any css asset applied ? In any case this consideration was not present in the commit message.

Reply:
> I don't get why we need to add `el` to an iframe and then add the iframe to the body. Why can't we add el to the body?
> 
> Isn't it an issue that your iframe does not have any css asset ? Or maybe it is added during `toInline` computation ? Will the computed style really be what we need without any css asset ? Or maybe it's the reason why you used the iframe in the first place, to not have any css asset applied ? In any case this consideration was not present in the commit message.

Yes, it is a better approach that directly append `el` to `document.body`, Instead of using 'iframe'. So I update the code accordingly. Thank you :)